nvidiaforceware.txt

```diff
@@ -0,0 +1,9 @@
+[Section]
+Name = NVIDIA Forceware driver
+Version = 181.22
+License = Unknown

```

Comment:
I guess that this would be the license
https://www.nvidia.com/en-us/drivers/geforce-license/